docker/exercise2/main.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+require 'faker'
+
+age = ARGV[0].to_i
```

Comment:
Handle the case when ARGV contains a string value
Worth having a check for the number of accepted arguments.